collect_app/src/main/java/org/odk/collect/android/utilities/ApplicationConstants.java

```diff
@@ -17,6 +17,8 @@
 
 public class ApplicationConstants {
 
+    public static final int SQLITE_MAXIMUM_QUERY_LIMIT = 999;
```

Comment:
Please add `// based on http://www.sqlite.org/limits.html` above so we have context for this constant.